src/main/java/org/jitsi/videobridge/octo/OctoChannel.java

```diff
@@ -177,6 +178,14 @@ public boolean setRtpEncodingParameters(
                         .getMediaStreamTracks())
                     .map(MediaStreamTrackDesc::getOwner)
                     .collect(Collectors.toSet()));
+
+            for (SourcePacketExtension s : sources)
+            {
+                if (MediaStreamTrackFactory.getOwner(s) == null)
+                {
+                    logger.warn("Received a source without an owner tag.");
```

Comment:
should we print some information about the s to be able to better debug if this happens?

Reply:
It should be pretty clear from the colibri xml that we log